ipn/ipnlocal/local_test.go

```diff
@@ -2173,3 +2175,199 @@ func TestOnTailnetDefaultAutoUpdate(t *testing.T) {
 		})
 	}
 }
+
+func TestSuggestDerpExitNode(t *testing.T) {
+	tests := []struct {
+		name       string
+		lastReport netcheck.Report
+		netMap     netmap.NetworkMap
+		wantValue  tailcfg.StableNodeID
+		wantError  error
+	}{
+		{
+			name: "returns exit node ID given exit nodes and latencies",
```

Comment:
The test case names should ideally be something sort that describes what's unique about that situation. I might call this one "2 nodes in same region".

It'll convert spaces to underscores, some people will claim that it shouldn't contain spaces so that searches for the failing test case name work better.